File: app/src/main/java/com/prosabdev/fluidmusic/ui/custom/CustomVerticalSeekBar.java
package com.prosabdev.fluidmusic.ui.custom;

import android.content.Context;
import android.util.AttributeSet;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import com.lukelorusso.verticalseekbar.VerticalSeekBar;

public class CustomVerticalSeekBar extends VerticalSeekBar {
    public CustomVerticalSeekBar(@NonNull Context context, @Nullable AttributeSet attrs) {
        super(context, attrs);
    }

    public CustomVerticalSeekBar(@NonNull Context context) {
        super(context);
    }

    public CustomVerticalSeekBar(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @Override
    public void setOnClickListener(@Nullable OnClickListener l) {
        super.setOnClickListener(l);
    }
}

File: app/src/main/java/com/prosabdev/fluidmusic/ui/custom/CustomVerticalSeekBar.kt
package com.prosabdev.fluidmusic.ui.custom

import android.content.Context
import android.util.AttributeSet
import com.lukelorusso.verticalseekbar.VerticalSeekBar

class CustomVerticalSeekBar : VerticalSeekBar {
    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs)
    constructor(context: Context) : super(context)
    constructor(context: Context, attrs: AttributeSet?, defStyleAttr: Int) : super(
        context,
        attrs,
        defStyleAttr
    )

    override fun setOnClickListener(l: OnClickListener?) {
        super.setOnClickListener(l)
    }
}
